libs/pilight/protocols/433.92/hideki.c

```diff
@@ -0,0 +1,509 @@
+/*
+	Copyright (C) 2018
+
+	This file is part of pilight.
```

Comment:
Please make this MPLv2

Reply:
New PR's and new code is slowly moving to MPLv2 because i believe GPLv3 is too restrictive. Also, i don't like how GPL forces me to make everything GPL.

Reply:
Do you have a MPLv2 header with copyright notice which I can use as template?